### PR TITLE
bridge: Smaller fixes to extend the step by step guide

### PR DESCRIPTION
**Pull request by fabiand**
- README changes
- Small error improvement
- Example bridge-consumer pod

Signed-off-by: Fabian Deutsch <fabiand@fedoraproject.org>

### CODE REVIEW

File: docs/network/bridge/README.md [hunk 1]
```diff
@@ -1,40 +1,57 @@
 # Network Bridge Device Plugin
 
-This device plugin allows user to attach pods to bridges running on nodes.
+This device plugin allows user to attach pods to existing bridges on nodes.
 
 ## Usage
 
-User must specify list of bridges that should be exposed to pods. Please note,
-that bridges must be manually precreated on all nodes.
+The user starts by ensuring that a bridg eexists on a given host.
```
Comment (phoracek) [inline docs/network/bridge/README.md]: bridge exists

File: docs/network/bridge/README.md [hunk 2]
```diff
@@ -1,40 +1,57 @@
...
+```bash
+$ ip link create mybr0 type bridge
+$ ip link set dev mybr0 up
+$ ip link show mybr0
+6: mybr0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
+    link/ether fe:1b:e5:82:d3:b5 brd ff:ff:ff:ff:ff:ff
 ```
-kubectl create configmap device-plugin-network-bridge --from-literal=bridges="br0,br1"
+
+Now the user must specify list of bridges that should be exposed to pods:
```
Comment (phoracek) [inline docs/network/bridge/README.md]: could be worth saying that multiple bridges can be passed as "br1,br2"

File: docs/network/bridge/README.md [hunk 3]
```diff
@@ -1,40 +1,57 @@
...
 
 ## Usage
 
-User must specify list of bridges that should be exposed to pods. Please note,
-that bridges must be manually precreated on all nodes.
+The user starts by ensuring that a bridg eexists on a given host.
+If it doesn't exit, then he can easily create it as a root as follows:
 
+```bash
+$ ip link create mybr0 type bridge
```
Comment (phoracek) [inline docs/network/bridge/README.md]: here you use $ to highlight command, in other examples you dont

**Review by phoracek — CHANGES_REQUESTED** [review]

File: docs/network/bridge/README.md [hunk 4]
```diff
@@ -1,40 +1,57 @@
...
 
-This device plugin allows user to attach pods to bridges running on nodes.
+This device plugin allows user to attach pods to existing bridges on nodes.
 
 ## Usage
 
-User must specify list of bridges that should be exposed to pods. Please note,
-that bridges must be manually precreated on all nodes.
+The user starts by ensuring that a bridg eexists on a given host.
+If it doesn't exit, then he can easily create it as a root as follows:
```
Comment (mpolednik) [inline docs/network/bridge/README.md]: This sentence sounds weird. I'd prefer

"If the bridge doesn't exist, the user can create it (with root privileges) as follows:"

File: docs/network/bridge/README.md (hunk 1 above)
Comment (fabiand) [reply docs/network/bridge/README.md]: thx

File: pkg/dpm/plugin.go [hunk 5]
```diff
@@ -107,6 +107,7 @@ func (dpi *DevicePlugin) register(kubeletEndpoint, resourceName string) error {
 	_, err = client.Register(context.Background(), reqt)
 	if err != nil {
 		glog.Errorf("Registration failed: %s", err)
+		glog.Errorf("Did you open the feature gate?")
```
Comment (mpolednik) [inline pkg/dpm/plugin.go]: Something less assuming such as "Make sure that the DevicePlugins feature gate is enabled" would be nicer. :)

File: docs/network/bridge/README.md (hunk 2 above)
Comment (fabiand) [reply docs/network/bridge/README.md]: done

File: docs/network/bridge/README.md (hunk 3 above)
Comment (fabiand) [reply docs/network/bridge/README.md]: highlighting it everywhere now, thanks

File: manifests/bridge-pod.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```
Comment (mpolednik) [inline manifests/bridge-pod.yml]: We need to make the distinction in manifests/ between examples and actual deployment-related manifests.

Comment (fabiand) [reply manifests/bridge-pod.yml]: done

File: pkg/dpm/plugin.go (hunk 5 above)
Comment (fabiand) [reply pkg/dpm/plugin.go]: done

**Comment by fabiand** [comment]
I need #12 to finalize this update

**Comment by mpolednik** [comment]
@fabiand #12 was just merged.

**Comment by fabiand** [comment]
Ready to merge

**Pull request merged**